src/classes/ ffTestStud.cls

```diff
@@ -0,0 +1,194 @@
+public with sharing class ffTestStud {
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.